x/oracle/keeper/oracle.go

```diff
@@ -88,33 +95,28 @@ func (k Keeper) ApproveOracleRegistration(ctx sdk.Context, msg *types.MsgApprove
 
 }
 
-// validateApproveOracleRegistration checks signature
-func (k Keeper) validateApproveOracleRegistration(ctx sdk.Context, msg *types.MsgApproveOracleRegistration) error {
-
+// validateApprovalSharingOracleKey validate unique ID of ApprovalSharingOracleKey and its signature
+func (k Keeper) validateApprovalSharingOracleKey(ctx sdk.Context, approval *types.ApprovalSharingOracleKey, signature []byte) error {
```

Comment:
This validation function is used in both `ApproveOracleRegistration` and `ApproveOracleUpgrade`.